Applications/Spire/Include/Spire/Ui/TextInputWidget.hpp

```diff
@@ -22,6 +22,9 @@ namespace Spire {
       */
       explicit TextInputWidget(QString text, QWidget* parent = nullptr);
 
+      //! Returns the left text padding.
+      int get_padding() const;
+
       //! Sets the input's style to the table cell style.
       void set_cell_style();
```

Comment:
Fix up these ```set_xxx_style``` methods:

    /** The styles available to render the TextInputWidget. */
    enum class Style {

      /** Render using the default style. */
      DEFAULT,

      /** Render using the table cell style. */
      CELL
    };

    void set_style(Style style);